server/routes/nuxt-api/__sitemap__/datasets.ts

```diff
@@ -0,0 +1,40 @@
+import { defineSitemapEventHandler } from '#imports'
+import type { SitemapUrl } from '#sitemap/types'
+
+export default defineSitemapEventHandler(async (event) => {
+  const config = useRuntimeConfig()
+
+  let { page = 1 } = getQuery(event)
+  page = parseInt(page)
+  const pageSize = 200
+  const nbSitemaps = 10 // should match the datasets sitemaps config in nuxt.config.tst
+
+  // computing starting api datasets page and max page for this sitemap page
+  let currentPage = null
```

Comment:
I think it's weird to start at null and not 1?

Reply:
Done in https://github.com/datagouv/cdata/pull/428/commits/5f5a9dabe70941a5767ce773d6d495ecc19258f1